pkg/providers/aws/cluster_network_provider.go

```diff
@@ -176,6 +177,29 @@ func (n *NetworkProvider) CreateNetwork(ctx context.Context, vpcCidrBlock *net.I
 		}
 		logger.Infof("creating vpc: %s", *createVpcOutput.Vpc.VpcId)
 
+		waitVPCerr := n.Ec2Api.WaitUntilVpcExists(&ec2.DescribeVpcsInput{
```

Comment:
nit: not important to change but if there are other changes the varialbe name would match other syntax used elsewhere.

```suggestion
		waitVpcErr := n.Ec2Api.WaitUntilVpcExists(&ec2.DescribeVpcsInput{
```